onnx/backend/test/case/node/split.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class Split(Base):
+
+    @staticmethod
+    def export_1d():
+        input = np.array([1., 2., 3., 4., 5., 6.])
+
+        node = onnx.helper.make_node(
+            'Split',
+            inputs=['input'],
+            outputs=['output_1', 'output_2', 'output_3'],
+            axis=0,
+            split=[2]
+        )
+
+        expected_outputs = [np.array([1, 2]), np.array([3, 4]), np.array([5, 6])]
+        expect(node, inputs=[input], outputs=[y for y in expected_outputs], name='test_split_equal_parts_1d')
+
+        node = onnx.helper.make_node(
+            'Split',
+            inputs=['input'],
+            outputs=['output_1', 'output_2'],
+            axis=0,
+            split=[2, 4]
+        )
+
+        expected_outputs = [np.array([1., 2.]), np.array([3., 4., 5., 6.])]
+        expect(node, inputs=[input], outputs=[y for y in expected_outputs], name='test_split_variable_parts_1d')
+
+     
+    @staticmethod
+    def export_2d():
+        input = np.array([[1., 2., 3., 4., 5., 6.],
+                          [7., 8., 9., 10., 11., 12.]])
+
+
+        node = onnx.helper.make_node(
+            'Split',
+            inputs=['input'],
+            outputs=['output_1', 'output_2'],
+            axis=1,
+            split=[3]
```

Comment:
We should not have the split here if we use equal split.

Reply:
Are you suggesting that the following part from defs.cc is not accurate too? 

"If the 'split' argument contains only one element, and that element divides the length of split axis evenly, the tensor is split into equal sized parts with given length."

Shouldn't it be the case that outputs length is determined by 'split' attribute ?

Reply:
Thank you very much! Lu.

I agree with Lu here. Sorry that I missed this point. "Split" should be either not available or containing same number of elements as number of outputs, and the sum of split elements should be equal to the dimension value along the axis.